app/scripts/views/mixins/resend-mixin.js

```diff
@@ -43,8 +43,8 @@ define([
 
     _updateResendButton: function () {
       var self = this;
-      // Hide the button after 4 attempts. Redisplay button after a delay.
-      if (self._attemptedSubmits === TOO_MANY_ATTEMPTS) {
+      // Hide the button after too many attempts. Redisplay button after a delay.
+      if (self._attemptedSubmits >= Constants.EMAIL_RESEND_MAX_TRIES) {
         self.logScreenEvent('too_many_attempts');
```

Comment:
I'm all for this, resend-mixin.js is "view" code, email limiting constants shouldn't belong in there. I'll update the PR this afternoon hopefully :)